storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -262,7 +262,7 @@
       (psim/kill-process thread-pid))
     (doseq [pid pids]
       (if as-user
-        (worker-launcher-and-wait conf user ["signal" pid "9"] :log-prefix (str "kill -15 " pid))
+        (worker-launcher-and-wait conf user ["signal" pid "9"] :log-prefix (str "kill -9 " pid))
```

Comment:
@caofangkun You are correct.  The JIRA title is wrong.

We want to send the SIGTERM, then wait, then send SIGKILL.